.github/workflows/test.yml

```diff
@@ -12,17 +12,6 @@ jobs:
   test:
     name: Full testing process
     runs-on: ubuntu-latest
-    permissions:
-      actions: read
-      contents: read
-      security-events: write
-      pull-requests: write
-      statuses: write
-
-    strategy:
-      fail-fast: false
-      matrix:
-        language: [ 'typescript' ]
```

Comment:
For some reason, the test reporter views in GitHub Actions were attaching themselves in the UI to the end of the CodeQL script run rather than this one. After removing this block, they started going to the correct place in the UI. ¯\_(ツ)_/¯